templates/next/pages/foos/[id]/edit.tsx

```diff
@@ -38,6 +40,28 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 export const getStaticPaths: GetStaticPaths = async () => {
   try {
     const response = await fetch("/{{{name}}}");
+    const view = response.data['{{{hydraPrefix}}}view'];
+    const paths = response.data["{{{hydraPrefix}}}member"].map(({{{lc}}}) => `${ {{~lc}}['@id'] }/edit`);
+  
+    if (view) {
+      try {
+        const {
+          '{{{hydraPrefix}}}last': last
+        } = view;
+        for (let page = 2; page <= parseInt(last.replace(/^\/{{{name}}}\?page=(\d+)/, '$1')); page++) {
+          paths.concat(
+            await fetch(`/{{{name}}}?page=${page}`).data["{{{hydraPrefix}}}member"].map(({{{lc}}}) => `${ {{~lc}}['@id'] }/edit`)
+          );
+        }
+      } catch (e) {
```

Comment:
I think we can remove this try/catch then.

Reply:
Maybe should we move this complex logic to a helper function to avoid code duplication?